docs/usage.rst

```diff
@@ -60,15 +60,18 @@ contain interesting physics information describing the dataset.
 
 If you would like to extract parts of metadata, for example to extract
 only the dataset title, or only the Global Tag information for CMS
-datasets, you can pipe the output with suitable command-line tools
-such as **jq**:
+datasets, you can use **--output-fields** command-line option.
 
 .. code-block:: console
 
-    $ cernopendata-client get-metadata --recid 1 | jq -S '.metadata.title'
-    "/BTau/Run2010B-Apr21ReReco-v1/AOD"
-    $ cernopendata-client get-metadata --recid 1 | jq -S '.metadata.system_details.global_tag'
-    "FT_R_42_V10A::All"
+    $ cernopendata-client get-metadata --recid 1 --output-fields title
+    {
+      "title" : "/BTau/Run2010B-Apr21ReReco-v1/AOD"
+    }
+    $ cernopendata-client get-metadata --recid 1 --output-fields system_details.global_tag
+    {
+      "global_tag": "FT_R_42_V10A::All"
```

Comment:
Users would want to get only the value, so that they can use `cernopendata-client` is shell scripts like:

```
global_tag=$(cernopendata-client get-metadata --recid $myrecid --output-fields 'system_details.global_tag')
```

If we always output JSON, then people would still need to use something like `jq` to parse the output.

Two options:

- either add an option to get values only (such as `--output-values`)
- or always output only values and do not allow to specify multiple fields (so that output is always just text); and let people who want to do more complex stuff could always use `jq`.

I guess the latter seems nicer, but we would have to rename the option `--output-values`.

Reply:
After discussing IRL, renaming to `--output-values` and getting value only. 👍

Reply:
(1) Actually we are getting the JSON value i.e. also with the string quotes... Perhaps we should get rid of those...

```console
$ cernopendata-client get-metadata --recid 1 --output-values system_details.global_tag
"FT_R_42_V10A::All"
```

I guess users would expect to see just the value, as mentioned in the #41 issue decription:

We could do:

- if output is number: nothing to do, print as is
- if string: strip quotes
- if list of dict: print as is

(2) Also, since we don't allow multiple values, please do:

- rename option from `--output-values` to `--output-value`
- amend help strings which currently still say "Comma-separated list of fields...", see `cernopendata-client get-metadata --help`